src/vs/workbench/contrib/terminal/browser/terminal.ts

```diff
@@ -584,3 +587,12 @@ export const enum LinuxDistro {
 	Fedora = 2,
 	Ubuntu = 3,
 }
+
+export type InstanceInfo = IInstanceInfo | null;
+
+export interface IInstanceInfo {
+	group: ITerminalGroup,
+	groupIndex: number,
+	instance: ITerminalInstance,
+	localInstanceIndex: number
```

Comment:
I think "local" here is a little misleading as we typically use that for terminals with the pty on the shared process